### PR TITLE
OCPBUGS#19978 Adding aws.amazon.com to firewall allowlist

### PR DESCRIPTION
**Pull request by bscott-rh**
Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OCPBUGS-19978

Link to docs preview:
https://67763--docspreview.netlify.app/openshift-enterprise/latest/installing/install_config/configuring-firewall#configuring-firewall_configuring-firewall
(Step 4)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/configuring-firewall.adoc [hunk 1]
```diff
@@ -101,6 +101,10 @@ You can use the wildcards `\*.quay.io` and `*.openshiftapps.com` instead of `cdn
 |Required to access Alibaba Cloud services and resources. Review the link:https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/sdk/endpoints/endpoints_config.go?spm=a2c4g.11186623.0.0.47875873ciGnC8&file=endpoints_config.go[Alibaba endpoints_config.go file] to determine the exact endpoints to allow for the regions that you use.
 
 .15+|AWS
+|`aws.amazon.com`
+|443,80
+|Required to access AWS service endpoints to generate manifests using `openshift-install`
```
Comment (yunjiang29) [inline modules/configuring-firewall.adoc:106]: @bscott-rh , I think we can keep it the same as `iam.amazonaws.com`: `Used to install and manage clusters in an AWS environment.` or `Required to access AWS services and resources.`

File: modules/configuring-firewall.adoc [hunk 2]
```diff
@@ -101,6 +101,10 @@ You can use the wildcards `\*.quay.io` and `*.openshiftapps.com` instead of `cdn
 |Required to access Alibaba Cloud services and resources. Review the link:https://github.com/aliyun/alibaba-cloud-sdk-go/blob/master/sdk/endpoints/endpoints_config.go?spm=a2c4g.11186623.0.0.47875873ciGnC8&file=endpoints_config.go[Alibaba endpoints_config.go file] to determine the exact endpoints to allow for the regions that you use.
 
 .15+|AWS
+|`aws.amazon.com`
+|443,80
```
Comment (adellape) [inline modules/configuring-firewall.adoc:105]: Looks like other spots in this table are putting a space in there:
```suggestion
|443, 80
```

Comment (bscott-rh) [reply modules/configuring-firewall.adoc:105]: Good catch, thanks

**Pull request merged**